content/docs/get-started/aws/_index.md

```diff
@@ -13,6 +13,14 @@ aliases: ["/docs/quickstart/aws/"]
 
 {{< cloud-intro "AWS" >}}
 
-Let's start with installing Pulumi.
+- Configure a Pulumi Community Edition account
```

Comment:
It wouldn't be one of my goals to create an account. I'd leave this out, as it could easily turn people away who actually might not mind creating an account as part of the process.

Reply:
👍